ricoconverter/ricoconverter-convert/src/test/resources/ead2rico/_15b_level_hasOrHadPart_hasOrHadConstituent/expected.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:rico="https://www.ica.org/standards/RiC/ontology#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:html="http://www.w3.org/1999/xhtml"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+    xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xml:base="http://data.archives-nationales.culture.gouv.fr/">
+    <rico:Record rdf:about="record/058220">
+      <rico:hasDocumentaryFormType rdf:resource="https://www.ica.org/standards/RiC/vocabularies/documentaryFormTypes#FindingAid"/>
+      <rico:describesOrDescribed rdf:resource="recordResource/top-058220"/>
+      <rico:describesOrDescribed rdf:resource="recordResource/top-058220"/>
+      <rico:hasInstantiation>
+         <rico:Instantiation rdf:about="instantiation/058220-i1">
+            <rico:isInstantiationOf rdf:resource="record/058220"/>
+            <dc:format xml:lang="en">text/xml</dc:format>
+            <rico:identifier>FRAN_IR_058220</rico:identifier>
+            <rico:hasOrHadHolder rdf:resource="agent/005061"/>
+            <rdfs:seeAlso rdf:resource="https://www.siv.archives-nationales.culture.gouv.fr/siv/IR/FRAN_IR_058220"/>
+         </rico:Instantiation>
+      </rico:hasInstantiation>
+    </rico:Record>
+    
+  <!-- *** archdesc elements *** -->
+
+    <!-- cas 1 : archdesc RecordSet -->
+    <rico:RecordResource rdf:about="recordResource/top-058220">
+    	<rico:isOrWasDescribedBy rdf:resource="record/058220"/>
+      <rdf:type rdf:resource="https://www.ica.org/standards/RiC/ontology#RecordSet"/>
+      <rico:hasRecordSetType rdf:resource="https://www.ica.org/standards/RiC/vocabularies/recordSetTypes#Series"/>
+      <rico:hasInstantiation>
+        <rico:Instantiation rdf:about="instantiation/top-058220-i1">
+          <rico:isInstantiationOf rdf:resource="recordResource/top-058220"/>
+        </rico:Instantiation>
+      </rico:hasInstantiation>
+      <rico:includesOrIncluded rdf:resource="recordResource/058220-a1" />
+    </rico:RecordResource>
+
+    <rico:RecordResource rdf:about="recordResource/058220-a1">
+        <rico:isOrWasIncludedIn rdf:resource="recordResource/top-058220"/> 
+        <rico:hasInstantiation>
+          <rico:Instantiation rdf:about="instantiation/058220-a1-i1">
+            <rico:isInstantiationOf rdf:resource="recordResource/058220-a1"/>
+            <rdfs:seeAlso rdf:resource="https://www.siv.archives-nationales.culture.gouv.fr/siv/UD/FRAN_IR_058220/a1"/>
+          </rico:Instantiation>
+        </rico:hasInstantiation>  
+    </rico:RecordResource>
+
+    <!-- cas 3 : archdesc Unknown -->
+    <rico:RecordResource rdf:about="recordResource/top-058220">
+    	<rico:isOrWasDescribedBy rdf:resource="record/058220"/>
+      <rico:hasInstantiation>
```

Comment:
Should there not be a
`<rdf:type rdf:resource='https://www.ica.org/standards/RiC/ontology#RecordSet'/>`?
It is an archdesc with no level, and line 1550 of ead2rico.xslt would generate the line above unless I am wrong

Reply:
This is related to my comment here : https://github.com/ArchivesNationalesFR/rico-converter/issues/10#issuecomment-1476404819
I took the assumption that the converter should not generate an rdf:type = RecordSet for archdesc without an explicit level attribute.
Note that the function you are referring to is not even called in the case the level attribute does not exist, since the template that assigns type match `@level`, which does not exist in that case.
I agree this is a bit misleading.

If you tell me you want an rdf:type = RecordSet for archdesc without a level, then I will adjust accordingly.